source/pattern-library/application-framework/login-page/index.md

```diff
@@ -1,12 +1,79 @@
+      <li>The<b> text only area </b>(optional), located to the right of the fields, is available for additional information such as an introductory message about your application or you may want to include information about the latest release of your product here - such as a version number. Apply text sparingly, do not overload this area with too much text.</li>
+      <li><b>Informational, warning and errors</b> <b>messages </b>are displayed below the application name and above the fields when necessary. If there are multiple strings, each message would appear on its own line and the area adjusts as needed in order to fit these messages. A couple examples of these messages include:
+        <ol style="list-style-type: lower-alpha;">
+          <li>An error is triggered if the user incorrectly types their username or password. In this event, after the information has been submitted, the value entered for the username and other optional fields remains populated however, the value for the password is discarded.
+            <a class="list-img-block" href="{{site.baseurl}}assets/img/login_error.png">
+              <img src="{{site.baseurl}}assets/img/login_error.png" alt="Login error" width="699" height="364" />
+            </a>
+          </li>
+          <li>A warning message is triggered if the user has the caps lock turned on. Once the caps lock is turned off, the warning should disappear.
+            <a class="list-img-block" href="{{site.baseurl}}assets/img/login_warningNew.png">
+              <img src="{{site.baseurl}}assets/img/login_warningNew.png" alt="Login warning" width="699" height="364" />
+            </a>
+          </li>
+          <li>An informational message is triggered if the user is timed out of the application. They will be returned to the login page, which will display an informational message explaining that the session timed out due to inactivity.
+            <a class="list-img-block" href="{{site.baseurl}}assets/img/login_info2.png">
+              <img src="{{site.baseurl}}assets/img/login_info2.png" alt="Login info" width="699" height="364" />
+            </a>
+          </li>
+        </ol>
+      </li>
+    </ol>
+  </div>
+  <div role="tabpanel" class="tab-pane" id="code">
+    <p class="tn-prototype tn-prototype-inverse">
+      <a href="https://rawgit.com/patternfly/patternfly/master/tests/login.html" target="_blank">
+        <img src="{{site.baseurl}}assets/img/login.png" alt="Login" width="841" />
+      </a>
+    </p>
+    <p class="reference-markup">
+      <a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="login-layout-markup" href="#login-layout-markup">Reference Markup</a>
+    </p>
+    <div class="collapse in" id="login-layout-markup">
+      <pre class="prettyprint">
 &lt;!DOCTYPE html&gt;

```

Comment:
There is a lot of markup here -- even more omitted from these diffs. Can we replace this with an include using the existing login.html test page?